code/datums/action.dm

```diff
@@ -441,7 +441,25 @@
 	name = "Use [target.name]"
 	button.name = name
 
+/datum/action/item_action/cult_dagger
+	name = "Draw Blood Rune"
+	desc = "Use the ritual dagger to create a powerful blood rune"
+	icon_icon = 'icons/mob/actions/actions_cult.dmi'
+	button_icon_state = "draw"
+	buttontooltipstyle = "cult"
+	background_icon_state = "bg_demon"
+
+/datum/action/item_action/cult_dagger/Grant(mob/M)
+	if(iscultist(M))
+		..()
+		button.screen_loc = "6:157,4:-2"
+		button.moved = "6:157,4:-2"
+	else
```

Comment:
... can't you just else return? i haven't checked action code in a while but if it doesn't call parent it probably wouldn't grant it at all.

Reply:
Just mimicking what i saw in another proc, wasn't sure stopping the "Grant" was enough.